### PR TITLE
Fixing jenkins agents.

### PR DESCRIPTION
**Pull request by jkotalik**

### CODE REVIEW

File: test/Common.FunctionalTests/AppHostConfig/IIS.config [hunk 1]
```diff
@@ -225,7 +225,7 @@
             <add name="AnonymousAuthenticationModule" image="%windir%\System32\inetsrv\authanon.dll" />
             <add name="RequestFilteringModule" image="%windir%\System32\inetsrv\modrqflt.dll" />
             <add name="CustomErrorModule" image="%windir%\System32\inetsrv\custerr.dll" />
-            <add name="AspNetCoreModule" image="%SystemRoot%\system32\inetsrv\aspnetcore.dll" />
+            <!-- <add name="AspNetCoreModule" image="%SystemRoot%\system32\inetsrv\aspnetcore.dll" /> -->
```
Comment (pakrym) [inline test/Common.FunctionalTests/AppHostConfig/IIS.config]: Remove

File: test/Common.FunctionalTests/OutOfProcess/HelloWorldTest.cs [hunk 2]
```diff
@@ -21,8 +22,8 @@ public HelloWorldTests(ITestOutputHelper output = null) : base(output)
 
         public static TestMatrix TestVariants
             => TestMatrix.ForServers(DeployerSelector.ServerType)
-                .WithTfms(Tfm.NetCoreApp22, Tfm.Net461)
-                .WithAllApplicationTypes()
+                .WithTfms(Tfm.NetCoreApp22, Tfm.Net461).WithAllAncmVersions()
```
Comment (pakrym) [inline test/Common.FunctionalTests/OutOfProcess/HelloWorldTest.cs]: Why is this required?

File: test/Common.FunctionalTests/OutOfProcess/HelloWorldTest.cs [hunk 3]
```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.Server.IntegrationTesting;
 using Microsoft.AspNetCore.Server.IntegrationTesting.IIS;
 using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Logging;
```
Comment (pakrym) [inline test/Common.FunctionalTests/OutOfProcess/HelloWorldTest.cs]: Revert

**Comment by jkotalik** [comment]
I'll merge this once the jenkins agents run... eventually <_<

**Comment by jkotalik** [comment]
One of the jenkins tests passed!

**Comment by pakrym** [comment]
Which one?

**Comment by jkotalik** [comment]
https://ci3.dot.net/job/aspnet_IISIntegration/job/release_2.2/job/windows-Configuration_Debug_prtest/256/console. Also note that the failures in appveyor tests have already been documented.
